app/backend/src/clj/teet/notification/notification_ion.clj

```diff
@@ -6,6 +6,13 @@
             [datomic.client.api :as d]
             [teet.notification.notification-db :as notification-db]))
 
+(defn days-param
+  []
+  (let [param (environment/config-value :notify :application-expire-days)]
+    (if (nil? param)
+      45
+      (Integer/parseInt param))))
```

Comment:
No need to parseInt here or check for nil. You can specify the default value and the function to parse with in the environment ssm definition:
`(->ssm [:notify :application-expire-days] 45 #(Integer/parseInt %))` 

Then the `config-value` will always return an integer directly.